### PR TITLE
fix(TMRX-1314): remove external link icon

### PR DESCRIPTION
**Pull request by ibrahim-kurhan**
Remove external link icon

[TMRX-1314](https://nidigitalsolutions.jira.com/browse/TMRX-1314)

[TMRX-1314]: https://nidigitalsolutions.jira.com/browse/TMRX-1314?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bibicollins1 — APPROVED** [review]

File: packages/ts-newskit/src/components/slices/lead-article/index.tsx [hunk 1]
```diff
@@ -103,22 +103,25 @@ export const LeadArticle = ({
     : imageTop
       ? { xs: 'editorialHeadline040', md: 'editorialHeadline030' }
       : 'editorialHeadline040';
+  const displayImageAtTop = imageTop || hideImage;
```
Comment (bibicollins1) [inline packages/ts-newskit/src/components/slices/lead-article/index.tsx:106]: Sorry approved but then just looking at this - would this then show the image even if it's got hideImage true? Just seems a little confusing?

File: packages/ts-newskit/src/slices/lead-story-1/index.tsx [hunk 2]
```diff
@@ -74,7 +74,12 @@ export const LeadStory1 = ({
       : 'editorialHeadline030'
   };
 
-  const marginTop = singleArticle
+  const modifedSingleArticle = {
```
Comment (yukiinasawa) [inline packages/ts-newskit/src/slices/lead-story-1/index.tsx:77]: `modifiedSingleArticle`? opposed to `modifedSingleArticle`

Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/slices/lead-story-1/index.tsx]: fixed

File: packages/ts-newskit/src/components/slices/lead-article/index.tsx (hunk 1 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/components/slices/lead-article/index.tsx]: you are right just changed the variable to `displayArticleVertical` does this help?

**Review by yukiinasawa — APPROVED** [review]

**Pull request merged**